### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by priolette**
- Added month chart with selection for chart type
- Reformat code

### CODE REVIEW

File: Energy Saver/Pages/Index.cshtml.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class IndexModel : PageModel
         public List<List<Taxes>>? Taxes { get; set; }
 
         [BindProperty]
-        public List<decimal> taxComparison { get; set; }
+        public List<decimal> TaxComparison { get; set; }
```
Comment (Leminode) [inline Energy Saver/Pages/Index.cshtml.cs:21]: good catch

File: Energy Saver/Pages/Statistics.cshtml.cs [hunk 2]
```diff
@@ -1,32 +1,23 @@
...
+using static Energy_Saver.Services.ChartService;
 
 namespace Energy_Saver.Pages
 {
     public class StatisticsModel : PageModel
     {
         private readonly IChartService _chartService;
+        public Chart? YearChart { get; set; }
+        public Chart? MonthChart { get; set; }
+        public List<List<Taxes>>? Taxes { get; set; }
```
Comment (Leminode) [inline Energy Saver/Pages/Statistics.cshtml.cs:18]: Could use List\<Taxes\> instead of List\<List\<Taxes\>\>

**Review by Leminode — APPROVED** [review]

**Review by Tauras-Rim — APPROVED** [review]
Good changes, the second chart is a nice addition

File: Energy Saver/Pages/Statistics.cshtml.cs (hunk 2 above)
Comment (priolette) [reply Energy Saver/Pages/Statistics.cshtml.cs:18]: Will refactor for next week

**Pull request merged**